devicetypes/sky-nie/min-smart-plug.src/min-smart-plug.groovy

```diff
@@ -383,4 +534,20 @@ private logDebug(msg) {
 
 private logTrace(msg) {
     log.trace "$msg"
+}
+
+private isButtonAvailable() {
+    if(device == null){
+        log.error "isButtonAvailable device = null"
+        return true
```

Comment:
Did You really mean to return `true` here if the device is null ?

Reply:
Because all preferences display items have been changed to fixed display, the return value of isButtonAvailable() only works when there is a device configuration. When device=null, whatever is returned is ok.